azurerm/internal/services/appplatform/tests/spring_cloud_service_resource_test.go

```diff
@@ -233,12 +288,17 @@ resource "azurerm_spring_cloud_service" "test" {
     }
   }
 
+  trace {
+    enabled                         = true
```

Comment:
(as above) if we use the presence of the block to infer this we can remove this:

```suggestion
```